### DISCUSSION TITLE
Unable to gain access despite being accepted into program.

### DISCUSSION THREAD

**Discussion by FishyBearBeer** [community/community#6940] [Copilot Conversations]
"You don't have access to GitHub Copilot." after installing the visual studio extension and signing into the account that got the email.

**Comment by davecheney** [comment]
@FishyBearBeer I see your invite is active. Can you pls tell me more about the problem.

**Reply by FishyBearBeer** [reply]
I installed the extension via Visual Studio Code, then allowed the extension to sign into my Github account. This first time, the account was of friends who hadn't signed out of my computer, so it denied access, I then proceeded to log out and into my account and found that the popup to sign in wouldn't show up, so I tried the same thing with Copilot Nightly and _my_ account, but I received that message.

Now the popup to sign in won't show for either extension.

**Reply by davecheney** [reply]
Please see https://github.com/github/feedback/discussions/6711

**Reply by cboozb** [reply]
I had the same problem. I paid my $100 for year. Downloaded the extension. Closed VS 2022. Then the vsix launched and installed. Re-opened. visual studio.  This launches a small adialog that says GitHub Copilot. A device code will be copied to the clipboard. Please paste this on the GitHub website to authorize GitHub.  THERE WAS NOTHING ON MYU CLIPBOARD. 
IN VISUAL STUDIO I NOW JUST GET THIS SMALL DIALOG THAT SAYS GITHUB COPILOT ACCESS IS DENIED.

**Comment by ricnader** [comment]
Same here. Already tried a million things.
